sycl/test/type_traits/type_traits.cpp

```diff
@@ -151,9 +151,14 @@ int main() {
   test_is_arithmetic<s::half2>();
 
   test_make_type_t<int, d::gtl::scalar_unsigned_int_list, unsigned int>();
-  test_make_type_t<s::cl_int, d::gtl::scalar_float_list, s::cl_float>();
-  test_make_type_t<s::cl_int3, d::gtl::scalar_unsigned_int_list, s::cl_uint3>();
-  test_make_type_t<s::cl_int3, d::gtl::scalar_float_list, s::cl_float3>();
+  test_make_type_t<s::opencl::cl_int, d::gtl::scalar_float_list,
+                   s::opencl::cl_float>();
+  // FIXME: is it a correct test change?
```

Comment:
What change?

This probably should be a comment in the review, not in the code. The context of this question is lost when PR is merged.